src/univariate/continuous/normalcanon.jl

```diff
@@ -59,6 +59,12 @@ entropy(d::NormalCanon) = (-log(d.λ) + log2π + 1) / 2
 location(d::NormalCanon) = mean(d)
 scale(d::NormalCanon) = std(d)
 
+function kldivergence(p::NormalCanon, q::NormalCanon)
+    pn = convert(Normal, p)
+    qn = convert(Normal, q)
+    return kldivergence(pn, qn)
```

Comment:
Did you check if it's more efficient to work with the alternative parameterization directly, without converting to Normal and using its implementation?

Reply:
No, I haven't checked that. You mean in terms of saving a few floating-point ops, or in terms of unwrapping those types?